### PR TITLE
rename `tidb_ddl_version to tidb_enable_fast_create_table`

### PR DESCRIPTION
**Pull request by GMHDBJD**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: fast-create-table.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: TiDB 加速建表
+summary: 介绍 TiDB 加速建表中的概念、原理、实现和影响。
+---
```
Comment (lilin90) [inline fast-create-table.md:4]:
```suggestion
aliases: ['/docs/dev/ddl-v2/']
---
```

File: basic-features.md [hunk 2]
```diff
@@ -130,7 +130,7 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
 | [元数据锁](/metadata-lock.md) |  Y  |  Y  | Y | Y | N | N | N | N | N | N | N |
 | [`FLASHBACK CLUSTER`](/sql-statements/sql-statement-flashback-cluster.md) |  Y  |  Y  | Y | Y | N | N | N | N | N | N | N |
 | [暂停](/sql-statements/sql-statement-admin-pause-ddl.md)/[恢复](/sql-statements/sql-statement-admin-resume-ddl.md) DDL | Y | Y | N | N | N | N | N | N | N | N | N |
-| [TiDB DDL V2 架构](/ddl-v2.md) | E | N | N | N | N | N | N | N | N | N | N |
+| [TiDB 加速建表](/fast-create-table.md) | E | N | N | N | N | N | N | N | N | N | N |
```
Comment (hfxsd) [inline basic-features.md:133]:
```suggestion
| [TiDB 加速建表](/fast-create-table.md) | E | E | N | N | N | N | N | N | N | N | N | N |
```

File: fast-create-table.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+# TiDB 加速建表
+
+从 v9.0.0 开始，TiDB 实现支持加速建表，可提升大批量建表的速度。
```
Comment (hfxsd) [inline fast-create-table.md:8]:
```suggestion
从 v8.0.0 开始，TiDB 实现支持加速建表，可提升大批量建表的速度。
```

File: fast-create-table.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+SET GLOBAL tidb_enable_fast_create_table = OFF;
+```
+
+## 实现原理
+
+TiDB 加速建表的实现步骤如下：
+
+1. 创建 `CREATE TABLE` Job。
+
+    此步骤和 V1 版本实现相同，通过解析 `CREATE TABLE` 语句生成相应的 DDL Job。
```
Comment (hfxsd) [inline fast-create-table.md:48]:
```suggestion
    通过解析 `CREATE TABLE` 语句生成相应的 DDL Job。
```

File: fast-create-table.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+2. 执行 `CREATE TABLE` Job。
+
+    与之前版本将 Job 插入 `mysql.tidb_ddl_job` 表中不同，新实现由接收该 `CREATE TABLE` 语句的 TiDB 节点直接执行建表语句，将表结构持久化到 TiKV 中。同时，将 `CREATE TABLE` Job 标记为已完成，插入到 `mysql.tidb_ddl_history` 表中。
```
Comment (hfxsd) [inline fast-create-table.md:52]:
```suggestion
    由接收该 `CREATE TABLE` 语句的 TiDB 节点直接执行建表语句，将表结构持久化到 TiKV 中。同时，将 `CREATE TABLE` Job 标记为已完成，插入到 `mysql.tidb_ddl_history` 表中。
```

File: system-variables.md [hunk 6]
```diff
@@ -1519,10 +1519,10 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 默认值：`1`
-- 取值范围：`1` 或 `2`
-- 这个变量用于控制是否开启 [TiDB DDL V2](/ddl-v2.md)。将该变量的值设置为 `2` 可以开启该功能，设置为 `1` 可以关闭该功能。默认值为 `1`。开启后，将使用 TiDB DDL V2 执行 DDL 语句。
-- 从 TiDB v7.6.0 开始，支持加速建表 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md)。
+- 默认值：`OFF`
+- 取值范围：`ON` 或 `OFF`
+- 这个变量用于控制是否开启 [TiDB 加速建表](/fast-create-table.md)。将该变量的值设置为 `ON` 可以开启该功能，设置为 `OFF` 可以关闭该功能。默认值为 `OFF`。
+- 从 TiDB v8.0.0 开始，支持加速建表 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md)。
```
Comment (hfxsd) [inline system-variables.md:1526]:
```suggestion
- 该变量在 v7.6.0 版本中名称为 [`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-%E4%BB%8E-v760-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5)
```

File: fast-create-table.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
+---
+title: TiDB 加速建表
+summary: 介绍 TiDB 加速建表中的概念、原理、实现和影响。
+aliases: ['/zh/tidb/dev/ddl-v2/']
```
Comment (tangenta) [inline fast-create-table.md:4]:
```suggestion
aliases: ['/zh/tidb/dev/fast-create-table/']
```

**Review by tangenta — APPROVED** [review]

File: system-variables.md [hunk 8]
```diff
@@ -1519,11 +1519,11 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 作用域：GLOBAL
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 默认值：`1`
-- 取值范围：`1` 或 `2`
-- 这个变量用于控制是否开启 [TiDB DDL V2](/ddl-v2.md)。将该变量的值设置为 `2` 可以开启该功能，设置为 `1` 可以关闭该功能。默认值为 `1`。开启后，将使用 TiDB DDL V2 执行 DDL 语句。
-- 从 TiDB v7.6.0 开始，支持加速建表 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md)。
-
+- 默认值：`OFF`
+- 取值范围：`ON` 或 `OFF`
```
Comment (hfxsd) [inline system-variables.md:1523]:
```suggestion
- 可选值：`ON`、`OFF`
```

File: system-variables.md [hunk 9]
```diff
@@ -1519,11 +1519,11 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
+- 这个变量用于控制是否开启 [TiDB 加速建表](/fast-create-table.md)。将该变量的值设置为 `ON` 可以开启该功能，设置为 `OFF` 可以关闭该功能。默认值为 `OFF`。
+- 从 TiDB v8.0.0 开始，支持加速建表 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md)。
+- 该变量在 v7.6.0 中名称为 [`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-%E4%BB%8E-v760-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5)。
```
Comment (hfxsd) [inline system-variables.md:1526]:
```suggestion
- 该变量在 v7.6.0 中名称为 [`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-%E4%BB%8E-v760-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5)。

```

**Review by hfxsd — APPROVED** [review]

File: fast-create-table.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+---
+
+# TiDB 加速建表
+
+从 v8.0.0 开始，TiDB 实现支持加速建表，可提升大批量建表的速度。
```
Comment (Oreoxmt) [inline fast-create-table.md:9]: `basic-features.md` 中是从 7.6.0 开始实验特性。

| 数据定义语言 (DDL) | 7.6 | 7.5 | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
| [TiDB 加速建表](/fast-create-table.md) | E | N | N | N | N | N | N | N | N | N | N |

File: releases/release-7.6.0.md [hunk 11]
```diff
@@ -269,7 +269,7 @@ TiDB 版本：7.6.0
 | [`tidb_auto_analyze_partition_batch_size`](/system-variables.md#tidb_auto_analyze_partition_batch_size-从-v640-版本开始引入) | 修改 | 经进一步的测试后，该变量默认值从 `1` 修改为 `128`。 |
 | [`tidb_sysproc_scan_concurrency`](/system-variables.md#tidb_sysproc_scan_concurrency-从-v650-版本开始引入) | 修改 | 在大规模集群里，`scan` 操作的并发度可以调整的更高，以满足 `ANALYZE` 的需要，因此将该变量最大值由 `256` 修改为 `4294967295`。 |
 | [`tidb_analyze_distsql_scan_concurrency`](/system-variables.md#tidb_analyze_distsql_scan_concurrency-从-v760-版本开始引入)       |    新增       |    用于设置执行 `ANALYZE` 时 `scan` 操作的并发度。默认值为 `4`。   |
-| [`tidb_ddl_version`](/system-variables.md#tidb_ddl_version-从-v760-版本开始引入)  |  新增  | 用于控制是否开启 [TiDB DDL V2](/ddl-v2.md)。将该变量的值设置为 `2` 可以开启该功能，设置为 `1` 关闭该功能。默认值为 `1`。开启后，将使用新版本的实现执行 DDL 语句。TiDB DDL V2 对 DDL 功能做了提升，建表 DDL 的执行速度相比 V1 版本提升 10 倍。 |
+| [`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-%E4%BB%8E-v760-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5)  |  新增  | 用于控制是否开启 [TiDB DDL V2](https://docs.pingcap.com/zh/tidb/v7.6/ddl-v2)。将该变量的值设置为 `2` 可以开启该功能，设置为 `1` 关闭该功能。默认值为 `1`。开启后，将使用新版本的实现执行 DDL 语句。TiDB DDL V2 对 DDL 功能做了提升，建表 DDL 的执行速度相比 V1 版本提升 10 倍。 |
```
Comment (Oreoxmt) [inline releases/release-7.6.0.md:272]:
```suggestion
| [`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-从-v760-版本开始引入)  |  新增  | 用于控制是否开启 [TiDB DDL V2](https://docs.pingcap.com/zh/tidb/v7.6/ddl-v2)。将该变量的值设置为 `2` 可以开启该功能，设置为 `1` 关闭该功能。默认值为 `1`。开启后，将使用新版本的实现执行 DDL 语句。TiDB DDL V2 对 DDL 功能做了提升，建表 DDL 的执行速度相比 V1 版本提升 10 倍。 |
```

File: system-variables.md [hunk 12]
```diff
@@ -1519,10 +1519,11 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 默认值：`1`
-- 取值范围：`1` 或 `2`
-- 这个变量用于控制是否开启 [TiDB DDL V2](/ddl-v2.md)。将该变量的值设置为 `2` 可以开启该功能，设置为 `1` 可以关闭该功能。默认值为 `1`。开启后，将使用 TiDB DDL V2 执行 DDL 语句。
-- 从 TiDB v7.6.0 开始，支持加速建表 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md)。
+- 默认值：`OFF`
+- 可选值：`ON`、`OFF`
+- 这个变量用于控制是否开启 [TiDB 加速建表](/fast-create-table.md)。将该变量的值设置为 `ON` 可以开启该功能，设置为 `OFF` 可以关闭该功能。默认值为 `OFF`。
+- 从 TiDB v8.0.0 开始，支持加速建表 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md)。
+- 该变量在 v7.6.0 中名称为 [`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-%E4%BB%8E-v760-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5)。
```
Comment (Oreoxmt) [inline system-variables.md:1526]:
```suggestion
- 该变量由 v7.6.0 [`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-从-v760-版本开始引入) 改名而来。
```

File: system-variables.md [hunk 13]
```diff
@@ -1519,10 +1519,11 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 作用域：GLOBAL
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 默认值：`1`
-- 取值范围：`1` 或 `2`
-- 这个变量用于控制是否开启 [TiDB DDL V2](/ddl-v2.md)。将该变量的值设置为 `2` 可以开启该功能，设置为 `1` 可以关闭该功能。默认值为 `1`。开启后，将使用 TiDB DDL V2 执行 DDL 语句。
-- 从 TiDB v7.6.0 开始，支持加速建表 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md)。
+- 默认值：`OFF`
+- 可选值：`ON`、`OFF`
+- 这个变量用于控制是否开启 [TiDB 加速建表](/fast-create-table.md)。将该变量的值设置为 `ON` 可以开启该功能，设置为 `OFF` 可以关闭该功能。默认值为 `OFF`。
```
Comment (Oreoxmt) [inline system-variables.md:1524]:
```suggestion
- 这个变量用于控制是否开启 [TiDB 加速建表](/fast-create-table.md)。
```

File: basic-features.md (hunk 2 above)
Comment (hfxsd) [inline basic-features.md:133]: 在另外一个 PR 里增加了 8.0.0 的内容 https://github.com/pingcap/docs-cn/pull/16796/files#r1523089400
```suggestion
| [TiDB 加速建表](/fast-create-table.md) | N | N | N | N | N | N | N | N | N | N | N |
```

File: fast-create-table.md (hunk 10 above)
Comment (hfxsd) [reply fast-create-table.md:9]: Fixed

File: system-variables.md [hunk 14]
```diff
@@ -1519,10 +1519,11 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 默认值：`1`
-- 取值范围：`1` 或 `2`
-- 这个变量用于控制是否开启 [TiDB DDL V2](/ddl-v2.md)。将该变量的值设置为 `2` 可以开启该功能，设置为 `1` 可以关闭该功能。默认值为 `1`。开启后，将使用 TiDB DDL V2 执行 DDL 语句。
-- 从 TiDB v7.6.0 开始，支持加速建表 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md)。
+- 默认值：`OFF`
+- 可选值：`ON`、`OFF`
+- 这个变量用于控制是否开启 [TiDB 加速建表](/fast-create-table.md)。
+- 从 TiDB v8.0.0 开始，支持加速建表 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md)。
```
Comment (hfxsd) [inline system-variables.md:1525]:
```suggestion
- 从 TiDB v8.0.0 开始，支持使用 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md) 加速建表。
```

File: system-variables.md [hunk 15]
```diff
@@ -1519,10 +1519,11 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 默认值：`1`
-- 取值范围：`1` 或 `2`
-- 这个变量用于控制是否开启 [TiDB DDL V2](/ddl-v2.md)。将该变量的值设置为 `2` 可以开启该功能，设置为 `1` 可以关闭该功能。默认值为 `1`。开启后，将使用 TiDB DDL V2 执行 DDL 语句。
-- 从 TiDB v7.6.0 开始，支持加速建表 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md)。
+- 默认值：`OFF`
+- 可选值：`ON`、`OFF`
+- 这个变量用于控制是否开启 [TiDB 加速建表](/fast-create-table.md)。
+- 从 TiDB v8.0.0 开始，支持使用 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md) 加速建表。
+- 该功能在 v7.6.0 中由 [`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-从-v760-版本开始引入) 控制。
```
Comment (hfxsd) [inline system-variables.md:1526]:
```suggestion
- 在 v7.6.0 中，该功能由 [`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-从-v760-版本开始引入) 控制。
```

File: fast-create-table.md (hunk 10 above)
Comment (Frank945946) [inline fast-create-table.md:9]: 并不是 8.0.0 开始的吧，其实 7.6 就引入了

File: system-variables.md [hunk 16]
```diff
@@ -1519,10 +1519,11 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 默认值：`1`
-- 取值范围：`1` 或 `2`
-- 这个变量用于控制是否开启 [TiDB DDL V2](/ddl-v2.md)。将该变量的值设置为 `2` 可以开启该功能，设置为 `1` 可以关闭该功能。默认值为 `1`。开启后，将使用 TiDB DDL V2 执行 DDL 语句。
-- 从 TiDB v7.6.0 开始，支持加速建表 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md)。
+- 默认值：`OFF`
+- 可选值：`ON`、`OFF`
+- 这个变量用于控制是否开启 [TiDB 加速建表](/fast-create-table.md)。
+- 从 TiDB v8.0.0 开始，支持使用 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md) 加速建表。
+- 在 v7.6.0 中，该功能由 [`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-从-v760-版本开始引入) 控制。
```
Comment (Frank945946) [inline system-variables.md:1526]:
```suggestion
- 从 TiDB v8.0.0 开始，支持使用 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md) 加速建表。该参数是由 v7.6.0 中，引入的 [`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-从-v760-版本开始引入) 更名而来。
```

Comment (Frank945946) [reply system-variables.md:1526]: @GMHDBJD 这次我们只是做了更名吧？而不是废弃`tidb_ddl_version`然后新增`tidb_enable_fast_create_table`，对么？

File: basic-features.md [hunk 17]
```diff
@@ -130,7 +130,7 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
 | [元数据锁](/metadata-lock.md) |  Y  |  Y  | Y | Y | N | N | N | N | N | N | N |
 | [`FLASHBACK CLUSTER`](/sql-statements/sql-statement-flashback-cluster.md) |  Y  |  Y  | Y | Y | N | N | N | N | N | N | N |
 | [暂停](/sql-statements/sql-statement-admin-pause-ddl.md)/[恢复](/sql-statements/sql-statement-admin-resume-ddl.md) DDL | Y | Y | N | N | N | N | N | N | N | N | N |
-| [TiDB DDL V2 架构](/ddl-v2.md) | E | N | N | N | N | N | N | N | N | N | N |
+| [TiDB 加速建表](/fast-create-table.md) | N | N | N | N | N | N | N | N | N | N | N |
```
Comment (Frank945946) [inline basic-features.md:133]: 这里改成 8.0 实验特性

Comment (hfxsd) [reply basic-features.md:133]: 已经在另外一个 PR 里改了 https://github.com/pingcap/docs-cn/pull/16796/files#r1523089400 
此处要保持现状，否则以后合入 PR 的时候会冲突。

File: system-variables.md (hunk 16 above)
Comment (GMHDBJD) [reply system-variables.md:1526]: 原变量已经不能用了

File: fast-create-table.md [hunk 18]
```diff
@@ -0,0 +1,57 @@
+---
+title: 提升 TiDB 建表性能
+summary: 介绍 TiDB 加速建表中的概念、原理、实现和影响。
+aliases: ['/zh/tidb/dev/ddl-v2/']
+---
+
+# 提升 TiDB 建表性能
+
+从 v7.6.0 开始，TiDB 实现支持加速建表，可提升大批量建表的速度。
```
Comment (hfxsd) [inline fast-create-table.md:9]:
```suggestion
TiDB v7.6.0 引入了系统变量 [`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-从-v760-版本开始引入) 实现支持加速建表，可提升大批量建表的速度。从 v8.0.0 开始，该系统变量更名为 [`tidb_enable_fast_create_table`](/system-variables.md#tidb_enable_fast_create_table-从-v800-版本开始引入)。
```

File: fast-create-table.md [hunk 19]
```diff
@@ -0,0 +1,57 @@
+---
+title: 提升 TiDB 建表性能
+summary: 介绍 TiDB 加速建表中的概念、原理、实现和影响。
+aliases: ['/zh/tidb/dev/ddl-v2/']
+---
+
+# 提升 TiDB 建表性能
+
+TiDB v7.6.0 引入了系统变量 [`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-从-v760-版本开始引入) 实现支持加速建表，可提升大批量建表的速度。从 v8.0.0 开始，该系统变量更名为 [`tidb_enable_fast_create_table`](/system-variables.md#tidb_enable_fast_create_table-从-v800-版本开始引入)。
```
Comment (Frank945946) [inline fast-create-table.md:9]:
```suggestion
TiDB v7.6.0 引入了系统变量 [`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-从-v760-版本开始引入) 实现支持加速建表，可提升大批量建表的速度。从 v8.0.0 开始，该系统变量更名为 [`tidb_enable_fast_create_table`](/system-variables.md#tidb_enable_fast_create_table-从-v800-版本开始引入)。[`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-从-v760-版本开始引入) 从 v8.0.0 开始不再生效，请使用新的变量名。
```

**Review by Frank945946 — APPROVED** [review]

File: fast-create-table.md (hunk 19 above)
Comment (hfxsd) [reply fast-create-table.md:9]: 8.0 的文档用相对链接即可。不用修改。

File: system-variables.md [hunk 20]
```diff
@@ -1519,10 +1519,11 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 默认值：`1`
-- 取值范围：`1` 或 `2`
-- 这个变量用于控制是否开启 [TiDB DDL V2](/ddl-v2.md)。将该变量的值设置为 `2` 可以开启该功能，设置为 `1` 可以关闭该功能。默认值为 `1`。开启后，将使用 TiDB DDL V2 执行 DDL 语句。
-- 从 TiDB v7.6.0 开始，支持加速建表 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md)。
+- 默认值：`OFF`
+- 可选值：`ON`、`OFF`
+- 这个变量用于控制是否开启 [TiDB 加速建表](/fast-create-table.md)。
+- 从 TiDB v8.0.0 开始，支持使用[`tidb_enable_fast_create_table`](/system-variables.md#tidb_enable_fast_create_table-从-v800-版本开始引入)加速建表。
+- 该变量是由 v7.6.0 中引入的 [`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-从-v760-版本开始引入) 更名而来。 [`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-从-v760-版本开始引入) 从 v8.0.0 开始不再生效。
```
Comment (hfxsd) [inline system-variables.md:1526]:
```suggestion
- 该变量是由 v7.6.0 中引入的 [`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-从-v760-版本开始引入) 更名而来。从 v8.0.0 开始，[`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-从-v760-版本开始引入) 不再生效。
```

File: system-variables.md [hunk 21]
```diff
@@ -1519,10 +1519,11 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 默认值：`1`
-- 取值范围：`1` 或 `2`
-- 这个变量用于控制是否开启 [TiDB DDL V2](/ddl-v2.md)。将该变量的值设置为 `2` 可以开启该功能，设置为 `1` 可以关闭该功能。默认值为 `1`。开启后，将使用 TiDB DDL V2 执行 DDL 语句。
-- 从 TiDB v7.6.0 开始，支持加速建表 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md)。
+- 默认值：`OFF`
+- 可选值：`ON`、`OFF`
+- 这个变量用于控制是否开启 [TiDB 加速建表](/fast-create-table.md)。
+- 从 TiDB v8.0.0 开始，支持使用[`tidb_enable_fast_create_table`](/system-variables.md#tidb_enable_fast_create_table-从-v800-版本开始引入)加速建表。
```
Comment (Oreoxmt) [inline system-variables.md:1525]:
```suggestion
- 从 TiDB v8.0.0 开始，支持使用 [`tidb_enable_fast_create_table`](/system-variables.md#tidb_enable_fast_create_table-从-v800-版本开始引入) 加速建表。
```

File: system-variables.md [hunk 22]
```diff
@@ -1519,10 +1519,11 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
+- 该变量是由 v7.6.0 中引入的 [`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-从-v760-版本开始引入) 更名而来。从 v8.0.0 开始，[`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-从-v760-版本开始引入) 不再生效。
```
Comment (Oreoxmt) [inline system-variables.md:1526]:
```suggestion
- 该变量是由 v7.6.0 中引入的 [`tidb_ddl_version`](https://docs.pingcap.com/zh/tidb/v7.6/system-variables#tidb_ddl_version-从-v760-版本开始引入) 更名而来。从 v8.0.0 开始，`tidb_ddl_version` 不再生效。
```

**Review by Oreoxmt — APPROVED** [review]

File: system-variables.md [hunk 23]
```diff
@@ -1519,10 +1519,11 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 作用域：GLOBAL
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 默认值：`1`
-- 取值范围：`1` 或 `2`
-- 这个变量用于控制是否开启 [TiDB DDL V2](/ddl-v2.md)。将该变量的值设置为 `2` 可以开启该功能，设置为 `1` 可以关闭该功能。默认值为 `1`。开启后，将使用 TiDB DDL V2 执行 DDL 语句。
-- 从 TiDB v7.6.0 开始，支持加速建表 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md)。
+- 默认值：`OFF`
```
Comment (Oreoxmt) [inline system-variables.md:1522]:
```suggestion
- 类型：布尔型
- 默认值：`OFF`
```

File: system-variables.md [hunk 24]
```diff
@@ -1519,10 +1519,11 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
+- 可选值：`ON`、`OFF`
```
Comment (Oreoxmt) [inline system-variables.md:1523]:
```suggestion
```

File: TOC.md [hunk 25]
```diff
@@ -982,7 +982,7 @@
         - [概述](/performance-schema/performance-schema.md)
         - [`SESSION_CONNECT_ATTRS`](/performance-schema/performance-schema-session-connect-attrs.md)
     - [元数据锁](/metadata-lock.md)
-    - [TiDB DDL V2](/ddl-v2.md)
+    - [TiDB 加速建表](/fast-create-table.md)
```
Comment (hfxsd) [inline TOC.md:985]:
```suggestion
    - [TiDB 加速建表](/accelerated-table-creation.md)
```

File: basic-features.md (hunk 17 above)
Comment (hfxsd) [inline basic-features.md:133]:
```suggestion
| [TiDB 加速建表](/accelerated-table-creation.md) | N | N | N | N | N | N | N | N | N | N | N |
```

File: system-variables.md [hunk 26]
```diff
@@ -1519,10 +1519,11 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 作用域：GLOBAL
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 默认值：`1`
-- 取值范围：`1` 或 `2`
-- 这个变量用于控制是否开启 [TiDB DDL V2](/ddl-v2.md)。将该变量的值设置为 `2` 可以开启该功能，设置为 `1` 可以关闭该功能。默认值为 `1`。开启后，将使用 TiDB DDL V2 执行 DDL 语句。
-- 从 TiDB v7.6.0 开始，支持加速建表 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md)。
+- 类型：布尔型
+- 默认值：`OFF`
+- 这个变量用于控制是否开启 [TiDB 加速建表](/fast-create-table.md)。
```
Comment (hfxsd) [inline system-variables.md:1524]:
```suggestion
- 这个变量用于控制是否开启 [TiDB 加速建表](/accelerated-table-creation.md)。
```

**Review by Oreoxmt — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

**Pull request merged**